fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/BrokerConfig.java

```diff
@@ -33,6 +33,9 @@ public class BrokerConfig extends ConfigBase {
     
     @ConfField
     public static int broker_ipc_port = 8000;
+
+    @ConfField
+    public static int kerberos_token_expire_seconds = 86000;
```

Comment:
Could you add comments for this new configuration and also inform the TW member to document for this.